requirements/nightly.in

```diff
@@ -1,11 +1,3 @@
-
+-r basetest.in
 # --- END OF CUSTOM SECTION ---
 # The following was generated by 'tox -e deps', DO NOT EDIT MANUALLY!
-dask
-graphviz
-pythreejs
-plopp @ git+https://github.com/scipp/plopp@main
-sciline @ git+https://github.com/scipp/sciline@main
-scippneutron @ git+https://github.com/scipp/scippneutron@main
-scippnexus @ git+https://github.com/scipp/scippnexus@main
-https://github.com/scipp/scipp/releases/download/nightly/scipp-nightly-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
```

Comment:
Forgot to run `tox -e deps` (or broken merge)?

Reply:
I ran it but the resulting .txt files only contained updates (no new or moved dependencies) So I decided against committing it in order to keep this PR more focussed.

Reply:
So how do you plan to fix the problem then?

Reply:
What problem?

Reply:
The problem that you removed all the nightly dependencies by running `copier update`, somehow?

Reply:
This is the `.in` file. The `.txt` file is unchanged. Next time, when someone runs `tox -e deps`, `nightly.in` will be filled back in.
But I restored the file anyway.